pom.xml

```diff
@@ -115,6 +137,26 @@
         </snapshotRepository>
     </distributionManagement>
 
+    <dependencies>
```

Comment:
JUnit, Hamcrest and Mockito are now defined once in the main `pom.xml` to prevent a Maven dependency problem between Hamcrast being part of JUnit and Mockito. This occurred with having several modules having those dependencies.